web/frontend/src/components/base.vue

```diff
@@ -121,15 +128,17 @@ const digestInput = () => {
   req.setName(props.name);
   req.setLinear(linear);
   req.setAngular(angular);
-
   rcLogConditionally(req);
```

Comment:
This is misleading - we are logging a request that never actually gets sent. We should add a hook to the SDK that allows clients to pass in a request logger.